include/deal.II/sundials/arkode.h

```diff
@@ -52,6 +52,9 @@ DEAL_II_NAMESPACE_OPEN
 // Shorthand notation for ARKODE error codes.
 #define AssertARKode(code) Assert(code >= 0, ExcARKodeError(code))
 
+/**
+ * A namespace for dealing with ODE solvers through the sundials package.
```

Comment:
Looking at their [documentation](https://computation.llnl.gov/projects/sundials), it would appear that `SUNDIALS` should be fully capitalised.